content/en/docs/setup/version-skew-policy.md

```diff
@@ -0,0 +1,112 @@
+---
+reviewers:
+- sig-api-machinery
+- sig-architecture
+- sig-cli
+- sig-cluster-lifecycle
+- sig-node
+- sig-release
+title: Kubernetes Version and Version Skew Support Policy
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+This document describes the maximum version skew supported between various Kubernetes components.
+Specific cluster deployment tools may place additional restrictions on version skew.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Supported versions
+
+The Kubernetes project maintains release branches for the most recent three minor releases,
```

Comment:
define what "minor" means. After rereading the doc, I guess it is the x in 1.x.y, not the y.

Reply:
I agree, maybe link to semantic versioning.

Reply:
> Kube X.Y.Z refers to the version (git tag) of Kubernetes that is released. This versions all components: apiserver, kubelet, kubectl, etc. (X is the major version, Y is the minor version, and Z is the patch version.)

for kubernetes, it would be (major, minor, patch).

Reply:
this wasn't non-obvious to me.

but for clarification this can be added somewhere near the top of the document:
"in this document X.Y.Z will refer to MAJOR.MINOR.PATCH"